### PR TITLE
Run Linter on Docker

### PR DESCRIPTION
**Pull request by tkatzir**
Added logs.

## Related Issues
fixes:

### CODE REVIEW

File: demisto_sdk/commands/lint/linter.py [hunk 1]
```diff
@@ -990,13 +990,22 @@ def _docker_image_create(self, docker_base_image: List[Any]) -> Tuple[str, str]:
 
     def _docker_remove_container(self, container_name: str):
         try:
+            logger.info(f"Trying to remove container {container_name}")
```
Comment (dorschw) [inline demisto_sdk/commands/lint/linter.py:993]:
```suggestion
            logger.debug(f"Trying to remove container {container_name}")
```

File: demisto_sdk/commands/lint/linter.py [hunk 2]
```diff
@@ -990,13 +990,22 @@ def _docker_image_create(self, docker_base_image: List[Any]) -> Tuple[str, str]:
...
             container = self._docker_client.containers.get(container_name)
+            logger.info(f"Found container {container_name=}, {container.id=}")
```
Comment (dorschw) [inline demisto_sdk/commands/lint/linter.py:995]:
```suggestion
            logger.debug(f"Found container {container_name=}, {container.id=}")
```

File: demisto_sdk/commands/lint/linter.py [hunk 3]
```diff
@@ -990,13 +990,22 @@ def _docker_image_create(self, docker_base_image: List[Any]) -> Tuple[str, str]:
...
             container.remove(force=True)
+            logger.info(f"Successfully removed container {container_name}")
```
Comment (dorschw) [inline demisto_sdk/commands/lint/linter.py:997]:
```suggestion
            logger.debug(f"Successfully removed container {container_name}")
```

File: demisto_sdk/commands/lint/linter.py [hunk 4]
```diff
@@ -990,13 +990,22 @@ def _docker_image_create(self, docker_base_image: List[Any]) -> Tuple[str, str]:
...
         except docker.errors.NotFound:
+            logger.info(f"Didn't remove container {container_name} as it wasn't found")
             pass
         except requests.exceptions.ChunkedEncodingError as err:
             # see: https://github.com/docker/docker-py/issues/2696#issuecomment-721322548
-            if platform.system() != "Darwin" or "Connection broken" not in str(err):
+            system_is_not_darwin = platform.system() != "Darwin"
+            not_connetion_broken = "Connection broken" not in str(err)
+            logger.info(
```
Comment (dorschw) [inline demisto_sdk/commands/lint/linter.py:1005]: either error or warning imo

**Review by dorschw — APPROVED** [review]

File: demisto_sdk/commands/lint/linter.py (hunk 1 above)
Comment (tkatzir) [reply demisto_sdk/commands/lint/linter.py:993]: These are usually info level. See line 1015

File: demisto_sdk/commands/lint/linter.py (hunk 2 above)
Comment (tkatzir) [reply demisto_sdk/commands/lint/linter.py:995]: These are usually info level. See line 1015

File: demisto_sdk/commands/lint/linter.py (hunk 4 above)
Comment (tkatzir) [reply demisto_sdk/commands/lint/linter.py:1005]: It's not necessarrily an error.
We try to remove the container anyway.
If it's not found - We're good.
But we want to know if it didn't find it, and then we'll get:
`Conflict. The container name "foo" is already in use by container "bar". You have to remove (or rename) that container to be able to reuse that name.`

**Pull request merged**